tests/migration_test.go

```diff
@@ -336,14 +336,14 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 
 			// verify it is numeric
 			_, err = strconv.Atoi(words[0])
-			Expect(err).ToNot(HaveOccurred(), "should have found pid for libvirtd that is numeric")
 
 			pid = words[0]
+			Expect(err).ToNot(HaveOccurred(), fmt.Sprintf(errorMassageFormat, stdout, stderr, errPsCmd, pid))
```

Comment:
This was previously above `pid = words[0]`, I think you should move it back there (if error is not nil there is no point to continue)

Reply:
done